ProjectShop/ProjectShop/urls.py

```diff
@@ -26,7 +26,8 @@
     path('admin/', admin.site.urls),
     path('users/', include('UserApp.urls')),
     path('pages/', include('django.contrib.flatpages.urls')),
-    path("api/", include("order.urls")),
+    path('api/v1/order/', include('order.urls')),
+    path('api/v1/products/', include(productRouter.urls)),
```

Comment:
watch my 121 Product App Interface via REST branch, I tried to resolve URL conflicts with routers